### ISSUE TITLE
Kerberos AS-REP etype 17/18 are no longer supported (disappeared?) in current version of hashcat

### ISSUE THREAD

**Issue by halfluke** [hashcat/hashcat#3926]
**Describe the bug**
After #2603  + #3729 I assumed hashcat would be able to crack AS-REP etype 18 for AS-REP roasting.
But either I cannot find the correct mode or the pull request has gone lost in translation at some point

**To Reproduce**
└─# hashcat --help | grep -i kerb                             
  19600 | Kerberos 5, etype 17, TGS-REP                              | Network Protocol
  19800 | Kerberos 5, etype 17, Pre-Auth                             | Network Protocol
  28800 | Kerberos 5, etype 17, DB                                   | Network Protocol
  19700 | Kerberos 5, etype 18, TGS-REP                              | Network Protocol
  19900 | Kerberos 5, etype 18, Pre-Auth                             | Network Protocol
  28900 | Kerberos 5, etype 18, DB                                   | Network Protocol
   7500 | Kerberos 5, etype 23, AS-REQ Pre-Auth                      | Network Protocol
  13100 | Kerberos 5, etype 23, TGS-REP                              | Network Protocol
  18200 | Kerberos 5, etype 23, AS-REP                               | Network Protocol

32200, 32100, or 33200, 33100 modes do not exist. Tried each of the other modes listed above always give signature invalid
Tried with John (see #2603 ), it worked
Tried with the hashcat fork from the pull request, switching to the add-aes-asrep-plugins branch (the one that was merged), recompiled hashcat and it works

**Expected behavior**
hashcat -m 32200, 32100 or 33200, 33100 should exist and be recognized as valid modes

**Hashcat version (please complete the following information):**
 - OS: Linux
 - Distribution: Kali
 - Version: 6.2.6

**Comment by Chick3nman** [comment]
Those modes were added after the 6.2.6 release, they will be in the next release but are currently only present in the repo and in the beta builds. If you would like to use them now, you will need to build hashcat from the master branch or grab a prebuilt beta binary from hashcat.net/beta, but with the understanding that beta builds are just that and may contain minor bugs.

I'm closing this as there doesn't seem to be an issue here, if you feel this has been closed in error, please feel free to reopen it.

**Comment by halfluke** [comment]
Ah ok thanks. Considering that the pull request was merged a long time ago, I thought they were included in the current release version, or that something had happened. Happy new year!

**Comment by Chick3nman** [comment]
Yeah, there's been a large gap between the last release and now, we have held off on pushing a new release because there are major changes coming. Happy new year!

**Comment by halfluke** [comment]
I checked so many things including compiling hashcat from the fork, and I didnt check if the additional modules were actually present in the current repo... oh well...
In the end I didnt even need it because the hash returned by kerbrute was type 18 but in a wrong format and it couldnt be cracked in any way, including by john. I opened this issue after testing the examples in #2603 
When I stopped insisting with kerbrute and used GetNPUsers.py, magically the hash returned was of type 23 and could be cracked with the classic mode 18200. Go figure... perhaps I should open another issue for kerbrute :-)
